packages/react-native-sdk/expo-config-plugin/src/withAndroidManifest.ts

```diff
@@ -46,8 +52,29 @@ const withStreamVideoReactNativeSDKManifest: ConfigPlugin = (configuration) => {
       );
       services.push(getNotifeeService());
       mainApplication.service = services;
+
+      if (props.androidPictureInPicture) {
+        const mainActivity = getMainActivityOrThrow(androidManifest);
+        ('keyboard|keyboardHidden|orientation|screenSize|uiMode');
```

Comment:
Does this do anything?

Reply:
What this says is.. "dont kill my screen even if these things change.. I can handle it "

it is the android docs to do this if PiP is to be enabled